tipsy-bot/tipsy.py

```diff
@@ -14,7 +15,6 @@
 # change this if you named your camera differently in your robot configuration
 camera_name = os.getenv("ROBOT_CAMERA") or "cam"
 # change this if you named your sensors differently in your robor configuration
-sensor_names = (os.getenv("ROBOT_SENSORS") or "ultrasonic,ultrasonic2").split(",")
```

Comment:
We should replace this with a variable for the service name to stay consistent with our other components:

```python
sensor_service_name = os.getenv("SENSOR_SERVICE") or "builtin"
```

Reply:
it's not something you configure. It's always builtin. You can't configure this service and give it a different name

Reply:
this should be:
sensor_service_name = (os.getenv("ROBOT_SENSORS") or "sensors")
you indeed can give it a different name as long as it refers to it in the code